cg/apps/lims/limsjson.py

```diff
@@ -57,14 +58,16 @@ def parse_json(indata: dict) -> dict:
     else:
         items = samples
 
-    data = {
+    order_data = {
         "customer": customer_id,
         "items": items,
```

Comment:
I think 'items' are either cases OR samples, do you have any suggestion?

Reply:
Do they need to be forced together under items? If they were added individually to the data dict their naming will be natural. Otherwise maybe "ids", given that it could either a sample_id or a case_id

Reply:
each item is either a "full" case dict or sample dict

Reply:
suggestions?